src/config/jwt.config.ts

```
@@ -0,0 +1,17 @@
+import { registerAs } from '@nestjs/config';
+//jwt config
+export default registerAs('jwt', () => {
+    const secret = process.env.JWT_SECRET || '';
```

ESLint

[error] Delete `··`

(prettier/prettier)